.env

```diff
@@ -7,6 +7,9 @@ BASE_STAMINA=2000
 THRESHOLD_PCT=0.6
 VOTING_STRATEGY=1
 VOTING_PREFERENCES=2
+AGENT_RANDOM_QUANTITY=100
+AGENT_TEAM2_QUANTITY=100
+AGENT_TEAM1_QUANTITY=0
 AGENT_RANDOM_QUANTITY=0
 AGENT_TEAM6_QUANTITY=100
```

Comment:
Refactor so that we don't end up with multiple entries

Reply:
Sorry for that, I'm fixing it.

Reply:
will do